python/b2gedmntuples_cff.py

```diff
@@ -520,6 +520,26 @@
 #### FOR SYSTEMATICS
 jetVarsForSys = (
     cms.PSet(
+        tag = cms.untracked.string("jecUncertainty"),
+        quantity = cms.untracked.string("userFloat('jecUncertainty')")
+        ),
+    cms.PSet(
+        tag = cms.untracked.string("PtResolution"),
+        quantity = cms.untracked.string("userFloat('PtResolution')")
+        ),
+    cms.PSet(
+        tag = cms.untracked.string("ScaleFactor"),
+        quantity = cms.untracked.string("userFloat('ScaleFactor')")
+        ),
+    cms.PSet(
+        tag = cms.untracked.string("ScaleFactorUp"),
+        quantity = cms.untracked.string("userFloat('ScaleFactorUp')")
+        ),
+    cms.PSet(
+        tag = cms.untracked.string("ScaleFactorDown"),
+        quantity = cms.untracked.string("userFloat('ScaleFactorDown')")
+        ),
+    cms.PSet(
```

Comment:
@jkarancs better to have the names of the scale factors as before, else it would be difficult to make out what variable the scale factor corresponds to: JERSF, JERSFUp, JERSFDown would be good.